.circleci/config.yml

```diff
@@ -14,5 +14,7 @@ jobs:
       - run: make check-file-headers
       - run: npm ci --legacy-peer-deps
       - run: node lib/bin/create-docker-databases.js
-      - run: make test-full
-
+      - run: mkdir ~/junit
+      - run: MOCHA_FILE=~/junit/test-results.xml make test-ci
```

Comment:
I am interesting in the recent feature of CircleCI "rerunning failed tests", for that we would need this step to be:

```
      - run: 
          name: Run tests
          command: |
            mkdir test-results
            TEST_FILES=$(circleci tests glob "test/**/*.js")
            echo "$TEST_FILES" | circleci tests run --command="xargs npx mocha --exit --reporter mocha-junit-reporter --reporter-options mochaFile=./test-results/junit.xml"
      - store_test_results:
          path: test-results
```

Reply:
:+1: that could be a good next step